examples/turning_pt_coord_diff_coupled.py

```diff
@@ -0,0 +1,190 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Wed Sep 11 17:38:57 2019
+
+@author: Wenyang Lyu and Shibabrat Naik
+"""
+
+import numpy as np
+import matplotlib.pyplot as plt
+from scipy.integrate import solve_ivp
+import math
+#from mpl_toolkits.mplot3d import Axes3D
+from scipy import optimize
+
+# This needs testing for installation 
+import sys
+sys.path.insert(0, './src/')
+import turning_point_coord_difference as tpcd
+import coupled_quartic_hamiltonian as coupled
+# This needs testing for installation 
+
+# import tpcd ### import module xxx where xxx is the name of the python file xxx.py 
+
+import matplotlib as mpl
+from matplotlib import cm
+
+mpl.rcParams['mathtext.fontset'] = 'cm'
+mpl.rcParams['mathtext.rm'] = 'serif'
+
+
+#%% Setting up parameters and global variables
+
+save_final_plot = True
+show_final_plot = False
+show_itrsteps_plots = False # show iteration of the UPOs in plots
+N = 4          # dimension of phase space
+omega=1.00
+EPSILON_S = 0.0 #Energy of the saddle
+alpha = 1.00
+beta = 1.00
+epsilon= 1e-1
+parameters = np.array([1,omega, EPSILON_S, alpha, beta,omega,epsilon])
+eqNum = 1  
+eqPt = tpcd.get_eq_pts(eqNum, coupled.init_guess_eqpt_coupled, \
+                                       coupled.grad_pot_coupled, parameters)
+
+
+#%%
+#deltaE_vals = [0.01, 0.1, 1.00, 2.0, 4.0]
+#xLeft = [0.0,-0.05,0.01,0.01,0.18]
+#xRight = [0.05,0.10,0.18,0.18,0.18]
+#linecolor = ['b','r','g','m','c']
+deltaE_vals = [0.1, 1.00]
+xLeft = [0.0,0.01]
+xRight = [0.05,0.18]
+linecolor = ['b','r']
+"""
+e is the total energy
+n is the number of intervals we want to divide
+n_turn is the nth turning point we want to choose.
+"""
+for i in range(len(deltaE_vals)):
+    
+    e = deltaE_vals[i]
+    n = 4
+    n_turn = 2
+    deltaE = e - parameters[2]
+    
+    """
+    Trial initial Condition s.t. one initial condition is on the LHS of the UPO and the 
```

Comment:
Don't use triple-quoted strings in the middle of normal code. Use comments for that.

Reply:
Done.